### PR TITLE
crud article

### PR DESCRIPTION
**Pull request by LongDT2002**
## Related Tickets
- [Feature #67980](https://edu-redmine.sun-asterisk.vn/issues/67980)
- Feature [#67982](https://edu-redmine.sun-asterisk.vn/issues/67982)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- Thêm mới form thêm/sửa trong file form.py. 
- Tạo function thêm sửa xoá article trong views.py, thêm mới các đường dẫn vào urls.py
- Tạo trang mới riêng cho việc hiện các bài viết cá nhân

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Review by huuson9 — APPROVED** [review]

**Comment by LongDT2002** [comment]
ready

File: CMS/home/views.py [hunk 1]
```diff
@@ -60,6 +60,50 @@ def articleList(request):
 
     return render(request, 'index.html', context)
 
+def ownArticleList(request):
```
Comment (huongnt-2545) [inline CMS/home/views.py:65]: check login require cho các action: own, new, update, delete article e nhé

**Comment by LongDT2002** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**